vars/sectionPromoteBuildToStage.groovy

```diff
@@ -44,6 +44,12 @@ def call(params) {
 
         pcr.callAround("${deploymentStage.label}:promotion") {
           acr.retagForStage(deploymentStage, dockerImage)
+          def resourceGroup = env.PTL_AKS_RESOURCE_GROUP
+          def clusterName = env.PTL_AKS_CLUSTER_NAME
+          def aksSubscription = env.AKS_PTL_SUBSCRIPTION_NAME
+          if (subscription != 'sandbox') {
```

Comment:
just checking this works on both sds and cft? i.e. sds isn't called sbox?

Reply:
Good spot, will check